scripts/run_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+INSTALL=`dirname "$0"`
```

Comment:
$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd) is better

Reply:
Why ?

Reply:
works with relative and absolute path

Reply:
OK